util/include/crypto_utils.hpp

```diff
@@ -0,0 +1,36 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+#pragma once
+
+#include <utility>
+#include <string>
+#include <memory>
+namespace concord::util {
+class Crypto {
+ public:
+  enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
```

Comment:
We have this enum in IKeyExchanger.hpp already. Can we remove it from there and leave only one here since it's a more suitable place?

Reply:
We can, but I would prefer to have this refactoring in a different PR